docs/REST API/rest-api.md

```diff
@@ -8566,7 +8568,11 @@ Add a software package to install on macOS, Windows, and Linux (Ubuntu) hosts.
 | install_script  | string | form | Command that Fleet runs to install software. If not specified Fleet runs [default install command](https://github.com/fleetdm/fleet/tree/f71a1f183cc6736205510580c8366153ea083a8d/pkg/file/scripts) for each package type. |
 | pre_install_query  | string | form | Query that is pre-install condition. If the query doesn't return any result, Fleet won't proceed to install. |
 | post_install_script | string | form | The contents of the script to run after install. If the specified script fails (exit code non-zero) software install will be marked as failed and rolled back. |
+| install | string | form | Install method. Options are `manual` (use Install software to install on each host manually) and `automatic` installed to each host on the team. Default is `manual` |
```

Comment:
Discussing with @RachelElysia, we both think `install_type` would be clearer, does that work @noahtalerman? cc @lucasmrod

Reply:
Hey @RachelElysia and @jacobshandling I see what y'all are saying here.

I'm on board w/ `install_type`. I updated the PR.

`*_type` seems to be consistent w/ other Fleet API endpoints. For example `label_membership_type` in "Add label": https://fleetdm.com/docs/rest-api/rest-api#add-label

@spokanemac what do they call this in Munki? Looks like they call it `INSTALL` in installomator: https://github.com/Installomator/Installomator/wiki/Configuration-and-Variables#install-behavior-force-installation

Reply:
Marking this conversation as resolved to clean up the PR.